### PR TITLE
docs: add docs to get notifications based on type

### PR DESCRIPTION
**Pull request by raviteja83**

### CODE REVIEW

File: docs/javascript/v2/features/notifications.mdx [hunk 1]
```diff
@@ -207,6 +207,74 @@ function Notification() {
...
+
+<Tabs id="notification-types" items={['JavaScript', 'React']} />
+
+<Tab id="notification-types-0">
+
+```jsx{14-44}
+import { hmsNotifications } from './hms';
+
+const unsubscribe = hmsNotifications.onNotification((notification) => {
+    if (!notification) {
```
Comment (triptu) [inline docs/javascript/v2/features/notifications.mdx:222]: do we need this here?

File: docs/javascript/v2/features/notifications.mdx [hunk 2]
```diff
@@ -207,6 +207,74 @@ function Notification() {
...
+        console.log('notification type', notification.type);
+        console.log('data', notification.data);
+
+    }, [notification]);
+
+    React.useEffect(() => {
+        if (!peerNotification) {
+            return;
+        }
+        // add a switch case or if/else depending on number of types passed in useHMSNotifications
```
Comment (triptu) [inline docs/javascript/v2/features/notifications.mdx:266]: comment can be removed

File: docs/javascript/v2/features/notifications.mdx [hunk 3]
```diff
@@ -207,6 +207,74 @@ function Notification() {
...
+
+```jsx{14-44}
+import { useHMSNotifications } from '@100mslive/react-sdk';
+
+function Notification() {
+    const notification = useHMSNotifications(HMSNotificationTypes.PEER_JOINED);
+    const peerNotification = useHMSNotifications(HMSNotificationTypes.PEER_JOINED);
+
+
+    React.useEffect(() => {
```
Comment (triptu) [inline docs/javascript/v2/features/notifications.mdx:252]: why two use effects?

Comment (raviteja83) [reply docs/javascript/v2/features/notifications.mdx:252]: to keep them separate. handling will be slightly different.

File: docs/javascript/v2/features/notifications.mdx (hunk 1 above)
Comment (raviteja83) [reply docs/javascript/v2/features/notifications.mdx:222]: yes, initially we return null.

File: docs/javascript/v2/features/notifications.mdx (hunk 3 above)
Comment (raviteja83) [reply docs/javascript/v2/features/notifications.mdx:252]: easy to copy paste depending on what is used.

Comment (triptu) [reply docs/javascript/v2/features/notifications.mdx:252]: what's the difference between them both are `useHMSNotifications(HMSNotificationTypes.PEER_JOINED)`, right?

File: docs/javascript/v2/features/notifications.mdx (hunk 1 above)
Comment (triptu) [reply docs/javascript/v2/features/notifications.mdx:222]: even in non react case?

**Review by triptu — APPROVED** [review]

File: docs/javascript/v2/features/notifications.mdx (hunk 1 above)
Comment (raviteja83) [reply docs/javascript/v2/features/notifications.mdx:222]: only in react I think. will update.

**Review by triptu — APPROVED** [review]

**Pull request merged**